website/src/pages/index.js

```diff
@@ -132,10 +132,10 @@ function HomeCallToAction() {
 function TwitterButton({accountName}) {
   return (
     <a
-      href={`https://twitter.com/intent/follow?screen_name=${accountName}&region=follow_link`}
-      target="_blank"
-      className="twitter-follow-button">
-      <div className="icon" />
+      className="twitter-follow-button"
```

Comment:
@huntie did you intend to remove this?

Reply:
Yup.
- `href` changes — managed by the embed script.
- `target="_blank"` — managed by the embed script.
- `<div className="icon" />` — removed along with corresponding CSS.